force-app/main/default/classes/Comments.cls

```diff
@@ -0,0 +1,27 @@
+ //List<Class__c> classes = [SELECT Id, Class__c, ]
```

Comment:
When submitting code for review, always take out the commented out code and debug statements.